hubgrep/frontend_blueprint/routes/search.py

```diff
@@ -72,9 +79,9 @@ def search():
     )
 
 
-def get_search_feedback(results_total: int) -> str:
+def get_search_feedback(results_total: int, time_search: float) -> str:
     """Get a readable message for how a search performed."""
     if results_total > 0:
-        return "Found {} matching repositories.".format(results_total)
+        return "Found {} matching repositories in {:.4f}s.".format(results_total, time_search)
```

Comment:
It would be more flexible to make the values available in the template, and control it there. Then I could truncate by default but still show the whole time on hover, like the dates. But that can be a different ticket if we want.

Reply:
oh right, you should use gettext() function so that its picked up when someone wants to add translations.

Reply:
I started doing this in a PR which will want to merge into this one, once im done